java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/PostgresValidator.java

```diff
@@ -271,7 +277,7 @@ private void validatePrivileges() throws SQLException {
     }
 
     private void validateTablePrivileges(boolean isSuperUser) throws SQLException {
-        if (isSuperUser) {
+        if (isSuperUser || isMultiTableShared) {
```

Comment:
Why skipping these checks when `isMultiTableShared`? (Same question for following modifications)

Reply:
`isMultiTableShared` denotes whether the properties to be validated is shared by multiple cdc tables. When creating a cdc source job, its value will be `true` and the `table.name` property is absent. So that we need to skip validate check for upstream table.